src/components/Create/components/pages/Payouts/PayoutsPage.tsx

```diff
@@ -113,3 +116,22 @@ export const PayoutsPage: React.FC = () => {
     </Form>
   )
 }
+
+const DisabledTooltip = (
+  props: SelectionCardProps & { fundingTargetDisabledReason: string },
+) => (
+  <Tooltip
+    title={
+      props.isDisabled ? (
+        <Trans>
+          {props.title} is disabled when <b>Funding Target</b> is{' '}
```

Comment:
Thinkin bout semantics of this. I was going to suggest `<strong>` but actually i think it might be more semantic to use `<i style={{'fontWeight': 500}}` (and no italic style)